### ISSUE TITLE
Pointer under transparency section?

### ISSUE THREAD

**Issue by pravdomil** [mdgriffith/elm-ui#304]
Is not that confusing?

https://github.com/mdgriffith/elm-ui/blob/7919c5e4f43741fc2d9b98a60fdd354fdb31a997/src/Element.elm#L117-L119

**Comment by dullbananas** [comment]
yes it's confusing. also noticed typo "will (stil) take up space"

**Comment by pravdomil** [comment]
https://github.com/mdgriffith/elm-ui/pull/306
